web/src/main/webapp/WEB-INF/classes/setup/sql/data/data-db-default.sql

```diff
@@ -695,6 +695,8 @@ INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('metada
 
 INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('metadata/batchediting/accesslevel', 'Editor', 0, 12020, 'n');
 
+INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('metadata/history/accesslevel', 'Editor', 0, 12021, 'n');
```

Comment:
I see this other setting:

```
INSERT INTO Settings (name, value, datatype, position, internal) VALUES ('system/metadata/history/enabled', 'false', 2, 9171, 'n');
```

To enable the history, maybe can be unified, either changing the new one to be in `system/metadata/...` or add a migration to move this one to `metadata/history/...`

Reply:
The system/... does not work very well on UI. I will try metadata/history/enable

Reply:
I have migrated all system/metadata/history/enabled to metadata/history/enabled including database migration script with and update statement, ui, and Java constants.